src/core/assert/base.js

```diff
@@ -46,7 +46,7 @@ export function assert(
   sentinel,
   shouldBeTruthy,
   opt_message = 'Assertion failed',
-  var_args
+  ...msgArgs
```

Comment:
This is like considerably slower. `…args` should never be used unless you explicitly need access to the array methods, or need to pass the arguments outside of the current invocation.